src/plugins/data_source_management/public/components/text_content/text_content.ts

```diff
@@ -163,7 +163,19 @@ export const DELETE_THIS_DATA_SOURCE = i18n.translate(
 export const NEW_PASSWORD_TEXT = i18n.translate(
   'dataSourcesManagement.editDataSource.newPassword',
   {
-    defaultMessage: 'New password',
+    defaultMessage: 'Updated password',
+  }
+);
+export const CONFIRM_NEW_PASSWORD_TEXT = i18n.translate(
```

Comment:
Ive called this out in some of the other MD PR's, but lets move the translate functions into the react components. That way we have realtime translations instead of static translations that only run during compile time.

Reply:
Understood, I will resolve it today in a different PR as many PRs will have conflicts if I do it now.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2543

Reply:
Thanks @mpabba3003!